doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -1359,6 +1359,5 @@ Documentation
 
   * The Device configuration guides section and moved its contents to :ref:`ug_app_dev`.
   * The Advanced building procedures page and moved its contents to the :ref:`building` page.
-  * nRF70 Series support is upstreamed to Zephyr, hence the documentation is removed from the |NCS|.
```

Comment:
Please remove this file and add a comment in PR - https://github.com/nrfconnect/sdk-nrf/pull/18192.